server/index.js

```diff
@@ -0,0 +1,15 @@
+import express from 'express';
+import users from './routes/users';
+import boards from './routes/boards';
+
+const app = express();
+const port = process.env.PORT || 3000;
+
+app.use('/users/', users);
+app.use('/boards/', boards);
+
+app.get('/', (req, res) => {
+    res.send('Hey!');
+})
+
+app.listen(port);
```

Comment:
Also add a callback for consoling that server is starts and the URL on which it is started

Reply:
Done